src/libbson/doc/bson_oid_compare_unsafe.rst

```diff
@@ -0,0 +1,17 @@
+:man_page: bson_oid_compare_unsafe
+
+bson_oid_compare_unsafe()
+=========================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+  static inline int
+  bson_oid_compare_unsafe (const bson_oid_t *oid1, const bson_oid_t *oid2);
+Description
+-----------
+
+Identical to :symbol:`bson_oid_compare()`, but performs no integrity checking.
+
```

Comment:
Remove extra trailing newline at end of file. Also applies to other instances in this PR.